var/spack/repos/builtin/packages/py-jaxlib/package.py

```diff
@@ -50,19 +50,19 @@ class PyJaxlib(PythonPackage, CudaPackage):
     def patch(self):
         self.tmp_path = tempfile.mkdtemp(prefix="spack")
         self.buildtmp = tempfile.mkdtemp(prefix="spack")
-        # triple quotes necessary because of a variety
-        # of other embedded quote(s)
         filter_file(
-            """f"--output_path={output_path}",""",
-            """f"--output_path={output_path}","""
-            """f"--sources_path=%s","""
-            """f"--nohome_rc'","""
-            """f"--nosystem_rc'",""" % self.tmp_path,
```

Comment:
I'm sure these single quotes were here for a good reason, but I'm not sure what that reason was. Going to try removing them and see if the build still passes.